MultiVersions/Forge/main/integrateddynamics/lang/zh_tw.json

```
@@ -0,0 +1,1873 @@
+
+    "aspect.integrateddynamics.write.integer.redstone": "紅石訊號",
+    "aspect.integrateddynamics.write.integer.redstone.info": "輸出給定強度的紅石訊號",
+    "aspect.integrateddynamics.write.integer.redstone.pulse": "脈衝紅石訊號",
+    "aspect.integrateddynamics.write.integer.redstone.pulse.info": "在給定的值變動時輸出一個特定強度的脈衝",
+
+    "_comment": "# World aspects",
+    "aspect.integrateddynamics.read.boolean.world.entity": "是否有實體",
+    "aspect.integrateddynamics.read.boolean.world.entity.info": "檢查目標空間是否有實體",
+    "aspect.integrateddynamics.read.boolean.world.mob": "是否有生物",
+    "aspect.integrateddynamics.read.boolean.world.mob.info": "檢查目標空間是否有生物",
+    "aspect.integrateddynamics.read.boolean.world.animal": "是否有動物",
+    "aspect.integrateddynamics.read.boolean.world.animal.info": "檢查目標空間是否有動物",
+    "aspect.integrateddynamics.read.boolean.world.player": "是否有玩家",
+    "aspect.integrateddynamics.read.boolean.world.player.info": "檢查目標空間是否有玩家",
+    "aspect.integrateddynamics.read.boolean.world.item": "是否有物品",
+    "aspect.integrateddynamics.read.boolean.world.item.info": "檢查目標空間是否有物品",
+    "aspect.integrateddynamics.read.boolean.world.weather.clear": "是否為晴天",
+    "aspect.integrateddynamics.read.boolean.world.weather.clear.info": "檢查天氣是否為晴天",
+    "aspect.integrateddynamics.read.boolean.world.weather.raining": "是否為雨天",
+    "aspect.integrateddynamics.read.boolean.world.weather.raining.info": "檢查天氣是否為晴天",
+    "aspect.integrateddynamics.read.boolean.world.weather.thunder": "是否為雷雨",
+    "aspect.integrateddynamics.read.boolean.world.weather.thunder.info": "檢查天氣是否為雷雨",
+    "aspect.integrateddynamics.read.boolean.world.isday": "是否為白天",
+    "aspect.integrateddynamics.read.boolean.world.isday.info": "檢查時間是否為白天",
+    "aspect.integrateddynamics.read.boolean.world.isnight": "是否為晚上",
+    "aspect.integrateddynamics.read.boolean.world.isnight.info": "檢查時間是否為晚上",
+
+    "aspect.integrateddynamics.read.integer.world.entity": "實體數量",
+    "aspect.integrateddynamics.read.integer.world.entity.info": "目標空間的實體數量",
+    "aspect.integrateddynamics.read.integer.world.raincountdown": "雨天冷卻",
+    "aspect.integrateddynamics.read.integer.world.raincountdown.info": "此世界在幾 tick 後會下雨",
+    "aspect.integrateddynamics.read.integer.world.ticktime": "刻時間",
+    "aspect.integrateddynamics.read.integer.world.ticktime.info": "此世界平均每 tick 的時間（ms）",
+    "aspect.integrateddynamics.read.integer.world.daytime": "每日時間",
+    "aspect.integrateddynamics.read.integer.world.daytime.info": "一天已經過了多久",
+    "aspect.integrateddynamics.read.integer.world.lightlevel": "亮度",
+    "aspect.integrateddynamics.read.integer.world.lightlevel.info": "目標位置的亮度",
+    "aspect.integrateddynamics.read.integer.world.players": "玩家數",
+    "aspect.integrateddynamics.read.integer.world.players.info": "目前在線的玩家數",
+
+    "aspect.integrateddynamics.read.double.world.tps": "TPS",
+    "aspect.integrateddynamics.read.double.world.tps.info": "此世界一秒的 tick 數",
```

參考原版將**tick**譯作**刻**
```suggestion
    "aspect.integrateddynamics.read.double.world.tps.info": "此世界一秒的刻數",
```